### PR TITLE
fix: fixed signing error message, logging and error handling

### PR DESCRIPTION
**Pull request by theblockstalk**

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -1,6 +1,5 @@
-// IMPORTANT: The following 3 packages should be imported in this order:
+// IMPORTANT: The following 2 packages should be imported in this order:
 import 'reflect-metadata';
-import './src/utils/logs';
```
Comment (theblockstalk) [inline App.tsx:3]: someting in this file was causing https://github.com/Tonomy-Foundation/Tonomy-ID/issues/1220. Removing the file solved the issue. The `log.ts` service was a nice to have but not worth having if we could not solve this error (which I could not)

File: src/components/TErrorModal.tsx [hunk 2]
```diff
@@ -101,7 +99,7 @@ export default function TErrorModal(props: TErrorModalProps) {
                 title={props.title ?? 'Something went wrong'}
                 iconColor={theme.colors.error}
             >
-                {/* {props.children} */}
+                {props.children}
```
Comment (sadiabbasi) [inline src/components/TErrorModal.tsx:102]: children is an optional parameter in the props... will it work if there is no props.children?

File: src/utils/chain/antelope.ts [hunk 3]
```diff
@@ -180,14 +181,16 @@ export class AntelopePrivateKey extends AbstractPrivateKey implements IPrivateKe
...
-
         try {
+            const transaction = await this.signTransaction(data);
+
             const receipt = await this.chain.getApiClient().v1.chain.push_transaction(transaction);
 
             return new AntelopeTransactionReceipt(this.chain, receipt, transaction);
         } catch (error) {
-            console.error('sendTransaction()', JSON.stringify(error, null, 2));
+            if (error?.message?.includes('Provided keys, permissions, and delays do not satisfy declared authorizations at')) {
```
Comment (sadiabbasi) [inline src/utils/chain/antelope.ts:191]: didn't this throw any code.. that will looks cleaner..

File: src/components/TErrorModal.tsx (hunk 2 above)
Comment (theblockstalk) [reply src/components/TErrorModal.tsx:102]: yes it seems to work if there are no children

**Pull request merged**